src/pages/Login.jsx

```
@@ -43,14 +75,71 @@ export default function Login() {
+                <path fill="#ffba08" d="M12 12h10v10H12z" />
+              </svg>
+              Microsoft
+            </button>
+          </div>
+          <div>
+            <button
+              type="button"
+              onClick={handleFacebookLogin}
+              className={"w-full flex justify-center items-center px-4 py-2 border border-gray-300 rounded-md shadow-sm bg-white text-sm font-medium text-gray-500 hover:bg-gray-50"}
+            >
+              <svg className="w-5 h-5 mr-2" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)